src/output/flat.rs

```diff
@@ -278,6 +282,15 @@ impl<'module, W: io::Write> Context<'module, W> {
                 }
                 _ => panic!("expected sequence, found {value:?}"),
             },
+            Format::RepeatFallback(_narrow, _wide) => match value {
+                Value::Seq(values) => {
+                    for _v in values {
+                        (); // FIXME
```

Comment:
I am slightly unclear on the semantics of `check_covered`, and so I am not sure how to implement this...